OpenRA.Mods.Common/Traits/SupportPowers/SupportPower.cs

```diff
@@ -228,7 +228,7 @@ public virtual void PlayLaunchSounds()
 
 			var toPlayer = isAllied ? localPlayer ?? Self.Owner : localPlayer;
```

Comment:
We bail early above if `localPlayer == null`. Thereforce `toPlayer` will always be `localPlayer`.